analyzers/src/SonarAnalyzer.Common/Rules/UseUnixEpochBase.cs

```diff
@@ -102,6 +109,17 @@ private bool IsEpochCtor(SyntaxNode node, SemanticModel model)
     private static bool IsLiteralAndEqualTo(SyntaxNode node, int value) =>
         node is TLiteralExpression literal && IsValueEqualTo(literal, value);
 
+    private static bool CheckAndGetTypeName(SyntaxNode node, SemanticModel model, out string name)
```

Comment:
NITPICK: You could return the string directly and  in the if check if  is {} typeName
```suggestion
    private static string CheckAndGetTypeName(SyntaxNode node, SemanticModel model)
```